src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -620,11 +620,11 @@ bool IsPlacementGroupWildcardResource(const std::string &resource_name) {
 
   // The length of {placement_group_id} is fixed, so we just need to check that if the
   // length and the pos of `_group_` match.
-  if (resource_name_view.size() > pattern.size() + 2 * PlacementGroupID::Size()) {
+  if (resource_name_view.size() < pattern.size() + 2 * PlacementGroupID::Size()) {
```

Comment:
looks like a bug? can we add a unit test for it?

Reply:
`test_placement_group_reschedule_when_node_dead ` can actually cover this function, but I also just added a unit test for it.